pubsub/streaming-analytics/README.md

```diff
@@ -0,0 +1,211 @@
+# Stream Cloud Pub/Sub with Cloud Dataflow
```

Comment:
Is this page intended to be the tutorial or is this documentation on Google Cloud DevSite? If the tutorial is the markdown, it might be a better idea to publish this in the Community repo.

Reply:
I should add region tags next. The tutorial is going to go here: https://cloud.google.com/pubsub/docs/pubsub-dataflow